docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -105,7 +104,7 @@ To configure a test:
 
    ![Configure a test](img/project-test-config.png)
 
-   - **A** **Failure Calculation** - Sets the failure condition used to run against the test result. You can use any function that operates on a column or multiple columns.
+   - **A** **Failure Calculation** - Sets the failure condition used to run against the test result. You can use `count()` or any function that operates on a column or multiple columns.
```

Comment:
I haven't chatted with the engineering folks or tried a function. 
Maybe good to give an example function, if we are saying "any function" ?

Reply:
I'll follow up with Haresh and make a new change later if needed. For now I removed mention of any function.